src/Files.App/Views/SettingsPages/Tags.xaml.cs

```diff
@@ -75,6 +75,7 @@ private void RenameTextBox_TextChanged(object sender, TextChangedEventArgs e)
 		{
 			var text = ((TextBox)sender).Text;
 			editingTag!.IsNameValid = IsNameValid(text);
+			editingTag!.CanCommit = IsNameValid(text) || (editingTag!.Tag.Name == text);
```

Comment:
I don't think we need two properties, can you merge these?

Reply:
I added it because I don't want it to show as invalid when the tag name hasn't been changed, but the commit button should be disabled because there's nothing to change. I don't know of a way to have the same logic in XAML else I wouldn't have added the property